.github/pull_request_template.md

```diff
@@ -2,4 +2,3 @@ Complete each step to get your pull request merged in. [Learn more about the wor
 - [ ] [Assign members of your team](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) to review the pull request. 
 - [ ] Wait for pull request status checks to complete. If there are problems, fix them until you see that [all status checks are passing](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fsymfony.com%2Fdoc%2F4.3%2F_images%2Fdocs-pull-request-symfonycloud.png&f=1&nofb=1). 
 - [ ] Wait until the pull request has been reviewed *and approved* by a teammate
```

Comment:
```suggestion
- [ ] Wait until the pull request has been reviewed *and approved* by a teammate
- [ ] After code reviews are approved and you determine this PR is ready to merge, select *Squash and Merge* button on this screen, leave the title and description to the default values, then merge the PR.
```

I do suggest keeping something in here for newcomers to our team.

Reply:
I like having that explicitly said, but does it make sense to have with a checkbox? That would mean developers would have to come back and check the box after the merge.

Reply:
I am OK with using checkboxes or not. From experience of using checkboxes with our team for a while now, they seem to work well where no one has expressed issues with them. I suggest we keep them as checkboxes and we can change it later when we encounter a problem.